applications/integrationtests/mechanics/feti/CMakeLists.txt

```diff
@@ -7,52 +7,45 @@ add_subdirectory(meshes)
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     foreach(test Feti FetiGradientDamage)
-        add_executable(${test} ${test}.cpp)
+        add_executable(${test} EXCLUDE_FROM_ALL ${test}.cpp)
         target_link_libraries(${test} Mechanics Math Base FetiLib)
         target_link_libraries(${test} Boost::mpi ${LAPACK_LIBRARIES})
-        target_link_libraries(${test} ${ANN_LIBRARIES} ${MPI_LIBRARIES})
-        if(ENABLE_VISUALIZE)
-            target_link_libraries(${test} Visualize)
-        endif()
-        if(MUMPS_FOUND)
-            target_link_libraries(${test} ${MUMPS_LIBRARIES})
-            target_link_libraries(${test} ${CMAKE_THREAD_LIBS_INIT})
-        endif()
+        target_link_libraries(${test} ${MPI_LIBRARIES})
```

Comment:
Is it advantageous to also convert MPI_LIBRARIES and LAPACK_LIBRARIES (and others...) into cmake targets? Is this manual coding or can we copy that from somewhere?

Reply:
I think it would be advantageous as well. This is mostly guesswork, reading the documentation and reading other FindXYZ modules, e.g. looking at FindPNG from CMake itself.